tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -167,6 +166,8 @@
         - install-certs
         - ovn
         - neutron-metadata
+        - configure-network
```

Comment:
just to note, that we might as well change this ordering in VAs, to keep things synced, if that makes sense for greenfield